src/index.html

```diff
@@ -7,6 +7,68 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day">
```

Comment:
Modifiers should be set by default.
<img width="848" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/54860641/9ee30d99-6c85-42d9-bf48-6ebe56150d96">

Reply:
> Modifiers should be set by default. <img alt="image" width="848" src="https://private-user-images.githubusercontent.com/54860641/302336983-9ee30d99-6c85-42d9-bf48-6ebe56150d96.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcyODc0MDksIm5iZiI6MTcwNzI4NzEwOSwicGF0aCI6Ii81NDg2MDY0MS8zMDIzMzY5ODMtOWVlMzBkOTktNmM4NS00MmQ5LWJmNDgtNmViZTU2MTUwZDk2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA3VDA2MjUwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZiYzQ1ODBlNDY4NDQyYWMxZmZmY2M0MGJiMDJlYTcwZGIyN2U0NTNjMTIyZTM5NjVlZGU3MDZlN2YzYmQxZjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.bgQ7wV1LUMmD-ZstwIyoFKF9CDz0tDauk5wOxzlXRVA">

Add default classes so a reviewer can test your solution, and add a demo link to the description in PR:
![image](https://github.com/mate-academy/layout_calendar/assets/63243136/6d296a97-6cc9-4301-b7b3-f79569faba31)